settlement/tests/treasury.rs

```diff
@@ -0,0 +1,234 @@
+//! An example of a settlement chain (Mythereum) abstracted in plain Rust code.
+//!
+//! When implementing a settlement treasury, this example will be helpful to understand the basic structure of message delivery.
+
+use light_client::LightClient;
+use merkle_tree::*;
+use simperby_common::merkle_tree::MerkleProof;
+use simperby_common::verify::CommitSequenceVerifier;
+use simperby_common::*;
+use simperby_settlement::execution::*;
+use std::cell::RefCell;
+use std::collections::HashMap;
+use std::rc::Rc;
+
+pub struct TetherContract {
```

Comment:
?What does Tether stand for?

Reply:
USDT

Reply:
usdt

Reply:
I thought it Test ether or something since other things were named Mythereum or so.